Updated Copyedit gRPC-Web plugin documentation (#3256)
Kong/docs.konghq.com: app/_hub/kong-inc/grpc-web/index.md

## Usage

> You should enable this plugin on a Kong Route that serves the `http(s)` protocol

Reviewer: ```suggestion
Enable this plugin on a Kong Route that serves the `http(s)` protocol
```

  --data name=grpc-web
```

> In these examples, there is no setting of configuration for the plugin.

Reviewer: ```suggestion
In these examples, we don't set any plugin configurations.
```

> This minimal setup works for the default varieties of the [gRPC-Web protocol],

Reviewer: ```suggestion
This minimal setup works for the default varieties of the [gRPC-Web protocol](https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-WEB.md#protocol-differences-vs-grpc-over-http2),
```

which use ProtocolBuffer messages either directly in binary or with base64-encoding.
The related `Content-Type` headers are `application/grpc-web` or `application/grpc-web+proto`
for binary, and `application/grpc-web-text` or `application/grpc-web-text+proto` for text.

If you want to use JSON encoding, you have to provide the gRPC specification in
a `.proto` file, which needs to be installed in the Kong node running the plugin.
A path starting with a `/` is considered absolute; otherwise, it interprets 

> relatively to the Kong node's prefix (`/usr/local/kong/` by default). For example:

Reviewer: ```suggestion
relative to the Kong node's prefix (`/usr/local/kong/` by default). For example:
```

  --data proto=path/to/hello.proto
```

> With this setup, there is support for gRPC-Web/JSON clients using `Content-Type` headers

Reviewer: ```suggestion
With this setup, there's support for gRPC-Web/JSON clients using `Content-Type` headers
```

like `application/grpc-web+json` or `application/grpc-web-text+json`.

> > **Note**: When using JSON encoding, the [gRPC-Web protocol] specifies that

Reviewer: ```suggestion
> **Note**: When using JSON encoding, the [gRPC-Web protocol](https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-WEB.md#protocol-differences-vs-grpc-over-http2) specifies that
```

both request and response data consist of a series of frames, in a similar way

> to the full [gRPC protocol]. The [gRPC-Web] library performs this framing as expected.

Reviewer: ```suggestion
to the full [gRPC protocol](https://github.com/grpc/grpc). The [gRPC-Web](https://github.com/grpc/grpc-web) library performs this framing as expected.
```
